examples/example_config.json

```diff
@@ -1,12 +1,27 @@
 {
-    "dataset_path": "https://ncsa.osn.xsede.org/Pangeo/pangeo-forge/noaa-coastwatch-geopolar-sst-feedstock/noaa-coastwatch-geopolar-sst.zarr",
-    "state": {
-        "active_array": "analysed_sst",
-        "x_array": "lat",
-        "x_scale": 2,
-        "y_array": "lon",
-        "t_array": "time",
-        "t_index": 5,
-        "main_drawer": false
+    "data_origin": "https://ncsa.osn.xsede.org/Pangeo/pangeo-forge/noaa-coastwatch-geopolar-sst-feedstock/noaa-coastwatch-geopolar-sst.zarr",
+    "data_array": {
```

Comment:
Thinking more about it. I don't like the name `data_array`. The reason is that it is not a data array but more of a volume or a mesh. 

I want the consistency between `dataset_builder.data_{xxxx}` and that entry `data_{xxxx}` in the config. But I find the current name misleading. 

Should we use `data_mesh` instead? @johnkit

Reply:
That does not need to be fixed for that PR. It is just a thought that should be addressed before we release.

Reply:
It may be worth considering that the dtype of the object returned for `dataset_builder.data_array` is an `xarray.DataArray`. While we typically use it as the source for a volume or mesh, a user could do something else with it if they desire. And the entry in the config includes all the information required to reconstruct that `xarray.DataArray` object.